test/Microsoft.AspNetCore.SystemWebAdapters.Tests/HttpContextTests.cs

```diff
@@ -152,8 +153,6 @@ public void GetServiceReturnsExpected()
             Assert.Same(context.Response, provider.GetService(typeof(HttpResponse)));
             Assert.Same(context.Server, provider.GetService(typeof(HttpServerUtility)));
             Assert.Same(context.Session, provider.GetService(typeof(HttpSessionState)));
-
-            Assert.Null(provider.GetService(typeof(HttpContext)));
```

Comment:
The code coverage is achieved via CacheTest.cs.

Reply:
It makes more sense to add the tests to a HttpContextTests class as the CacheTest shouldn't actually be testing that